src/libostree/ostree-sysroot-deploy.c

```diff
@@ -1797,14 +1798,35 @@ parse_os_release (const char *contents, const char *split)
   return ret;
 }
 
+static guint
+bootloader_get_max_boot_tries (OstreeSysroot *self, GCancellable *cancellable, GError **error)
+{
+  g_autoptr (OstreeRepo) repo = NULL;
+  if (!ostree_sysroot_get_repo (self, &repo, cancellable, error))
+    return 0;
+
+  return repo->boot_counting;
```

Comment:
This function doesn't adhere to the GError conventions (I know, it's messy) - or at least the caller down below just ignores a return value of `0` and continues (potentially leaving the `error` set).

But I think there's a simple solution; we don't need this wrapper function at all. It's safe to just deref from `self->repo->boot_counting` in callers (if the sysroot is loaded).

Reply:
removed this function